### PR TITLE
chore: move billing celery job to use offline cluster and increase timeout from 2 minutes to 5 min

### PR DESCRIPTION
**Pull request by fuziontech**
## Problem

We had some issues with the billing reporting timing out because the queries were taking longer than 120 seconds. They were also running on the online vs offline cluster. This moves the queries to the offline cluster and bumps the timeout to 5 minutes.

We may need to bump the timeout even higher.

Also adds retry logic for the queries.

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: posthog/clickhouse/client/execute.py [hunk 1]
```diff
@@ -40,13 +40,7 @@
...
 @lru_cache(maxsize=1)
 def default_settings() -> Dict:
-    # On CH 22.3 we need to disable optimize_move_to_prewhere due to a bug. This is verified fixed on 22.8 (LTS),
-    # so we only disable on versions below that.
-    # This is calculated once per deploy
-    if clickhouse_at_least_228():
-        return {"join_algorithm": "direct,parallel_hash", "distributed_replica_max_ignored_errors": 1000}
-    else:
-        return {"optimize_move_to_prewhere": 0}
+    return {"join_algorithm": "direct,parallel_hash", "distributed_replica_max_ignored_errors": 1000}
```
Comment (raquelmsmith) [inline posthog/clickhouse/client/execute.py:43]: @fuziontech just making sure this was intentional since it doesn't seem to be related

**Review by raquelmsmith — APPROVED** [review]

File: posthog/clickhouse/client/execute.py (hunk 1 above)
Comment (fuziontech) [reply posthog/clickhouse/client/execute.py:43]: it is - we are way beyond 22.8 at this point so this was no longer needed at all

Comment (fuziontech) [reply posthog/clickhouse/client/execute.py:43]: Honestly once I have a bit of time I need to go through all of the CH client settings and clean up everything. There's a lot of built up crud in here from different CH versions
